### PR TITLE
Added new Python tutorials

### PR DESCRIPTION
**Pull request by Zen-The-Tribe**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Zen-The-Tribe** [comment]
Adding new tutorials

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: more/free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -375,6 +375,8 @@
 * [Python Tutorial](https://www.w3schools.com/python) - W3Schools
 * [Python Tutorial](https://www.scaler.com/topics/python/) - Scaler Topics
 * [Scientific Computing with Python Certification](https://www.freecodecamp.org/learn/scientific-computing-with-python/) - freeCodeCamp
+* [Learn Python the Hard Way](https://learnpythonthehardway.org/python3/) - Learn Python the Hard Way
+* [A Byte of Python](https://python.swaroopch.com/) - Byte of Python
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-en.md:379]: duplicate

File: more/free-programming-interactive-tutorials-en.md [hunk 2]
```diff
@@ -375,6 +375,8 @@
 * [Python Tutorial](https://www.w3schools.com/python) - W3Schools
 * [Python Tutorial](https://www.scaler.com/topics/python/) - Scaler Topics
 * [Scientific Computing with Python Certification](https://www.freecodecamp.org/learn/scientific-computing-with-python/) - freeCodeCamp
+* [Learn Python the Hard Way](https://learnpythonthehardway.org/python3/) - Learn Python the Hard Way
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-en.md:378]: it's a book, and not free

**Pull request closed without merging**